### PR TITLE
tflint: update to 0.45.0, adopt

### PR DESCRIPTION
**Pull request by Emru1**
#### Testing the changes
- I tested the changes in this PR: **YES**

Builded and tested on amd64-glibc

### CODE REVIEW

**Comment by mhmdanas** [comment]
Append ", adopt" to the PR title and commit message.

File: srcpkgs/tflint/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+version=0.45.0
 revision=1
 build_style=go
 go_import_path="github.com/terraform-linters/$pkgname"
 short_desc="Terraform linter focused on possible errors, best practices, etc"
-maintainer="Sora Morimoto <sora@morimoto.io>"
+maintainer="Emil Tomczyk <emil.tomczyk@skni.umcs.pl>"
 license="MPL-2.0"
 homepage="https://github.com/terraform-linters/tflint"
 distfiles="${homepage}/archive/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/tflint/template:11]:
```suggestion
 distfiles=https://github.com/terraform-linters/tflint/archive/v${version}.tar.gz"
```

File: srcpkgs/tflint/template [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'tflint'
 pkgname=tflint
-version=0.22.0
+version=0.45.0
 revision=1
 build_style=go
 go_import_path="github.com/terraform-linters/$pkgname"
 short_desc="Terraform linter focused on possible errors, best practices, etc"
-maintainer="Sora Morimoto <sora@morimoto.io>"
+maintainer="Emil Tomczyk <emil.tomczyk@skni.umcs.pl>"
```
Comment (classabbyamp) [inline srcpkgs/tflint/template:8]: @smorimoto can they adopt this from you?

File: srcpkgs/tflint/template [hunk 3]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'tflint'
 pkgname=tflint
-version=0.22.0
+version=0.45.0
 revision=1
 build_style=go
 go_import_path="github.com/terraform-linters/$pkgname"
```
Comment (classabbyamp) [inline srcpkgs/tflint/template:6]:
```suggestion
 go_import_path="github.com/terraform-linters/tflint"
```

File: srcpkgs/tflint/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/tflint/template:11]: the hazards of mobile editing, but i think the point comes across

File: srcpkgs/tflint/template (hunk 2 above)
Comment (smorimoto) [reply srcpkgs/tflint/template:8]: I don't have any problem with it! Rather, I'm sorry that I haven't been able to take care of this for a long time! 😱

**Pull request merged**